web/package.json

```diff
@@ -19,22 +19,16 @@
     "build:integreat-test-cms": "yarn run webpack-ts --env config_name=integreat-test-cms",
     "build:obdach": "yarn run webpack-ts --env config_name=obdach",
     "check:build": "yarn es-check es2020 ./dist/**/*.js --verbose",
-    "test": "jest",
-    "lint": " yarn stylelint && yarn eslint --cache --cache-location ../.eslintcache .",
-    "stylelint": "stylelint --config stylelint.config.js \"src/**/*.{ts,tsx,css}\"",
```

Comment:
So running `lint` in the web dir/workspace will not automatically exeute `stylelint` anymore? Did you adjust the CI?